README.md

```diff
@@ -9,7 +9,8 @@ The **Otter** project is a highly modular framework whose goal is to provide a c
 It is split into several units to cover different aspects of these applications (localization, testing, customization, etc.).
 Also, to customize an application, metadata can be extracted from the application source code and injected into a CMS to manage dynamic configuration.
 
-> **Note**:  The full documentation is available [here](./docs/README.md).
+> **Note**: The full documentation is available [here](./docs/README.md).
+> **Note**: A showcase to help you integrate some simple use case is available [here](https://amadeusitgroup.github.io/otter/#/home) 
```

Comment:
Provide actually the link to the repo in the showcase could be a good idea also :D

Reply:
![image](https://github.com/AmadeusITGroup/otter/assets/110408057/48d4a4c4-b640-40f9-ab14-68a280c99472)